integration/testrunner/grpcrunner/grpc.go

```diff
@@ -73,32 +73,34 @@ var (
 // TestGRPCEndpoint runs tests against a GRPC endpoint. The given GRPC must
 // define a projectID and datasetQuery. If a TalkerClient or BigQueryClient are
 // not provided, they are instantiated via the defaults.
-func TestGRPCEndpoint(ctx context.Context, tb testing.TB, g *GRPC) {
-	tb.Helper()
+//
+//nolint:tparallel // t.Parallel() is already called
+func TestGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
```

Comment:
If we name this lowercase testGRPCEndpoint, then you don't need the nolint. Test helpers should also accept testing.TB, not *testing.T.

Reply:
(Throughout)

Reply:
> This is in a different package so the func must be exposed. How about `CheckGRPCEndpoint`?

I see, so we call this from other packages? Can it be inlined instead?